### DISCUSSION TITLE
How to mock react-native-dotenv in jest

### DISCUSSION THREAD

**Discussion by snab88** [goatandsheep/react-native-dotenv#204] [Q&A]
Hi, I am trying to mock `react-native-dotenv` but the values used in the tests are not the mocked ones but the actual env ones. 
I am mocking it in my tests as:

```
jest.mock('react-native-dotenv', () => ({
   ENV_VAR: 'mock'
}));

```

**Comment by goatandsheep** [comment]
my suggestion is to make a `.env.test`

it should work out of the box.
